WcaOnRails/app/views/polls/vote.html.erb

```diff
@@ -0,0 +1,31 @@
+<% provide(:title, 'Vote in poll #' + @poll.id.to_s) %>
+
+<div class="container">
+
+  <% if @poll.poll_is_over? %>
+    <div class="alert alert-danger">Poll is closed</div>
+  <% else %>
+
+  <h3><%= @poll.question %></h3>
+
+  <%= simple_form_for @vote do |f| %>
+
+    <%= f.input :poll_option_id, label: false, collection: @poll.poll_options, as: :radio_buttons, label_method: lambda { |c| c.description }, value_method: lambda { |c| c.id } %>
+
+    Comment:
+    <%= f.text_area :comment, rows: 3, required: false %>
```

Comment:
You mean if I take out the "Comment:" text?
Then nothing appears next to the text area...